docs/lib/storage/fragments/ios/configureaccess.md

```diff
@@ -16,7 +17,3 @@ Another user that wants to read the file can specify the user that created it:
 let options = StorageDownloadDataRequest.Options(accessLevel: .protected,
                                                  targetIdentityId: "OtherUserId")
 ```
-
-## Private Access
```

Comment:
are we removing these sections altogether?

Reply:
looks like the sections were empty? under this "configure file access levels" section, i think it's important to have them like. 
here's the JS equivalant in the old docs https://aws-amplify.github.io/docs/js/storage#file-access-levels
In the new JS docs: https://docs.amplify.aws/lib/storage/configureaccess?platform=js

we should just use the same for consistency, even though i recall in one of the naming meetings, we decided to rename "public" to "guest".

Reply:
We should have them indeed but it's out of the scope of this PR. Those sections are non-existent in the [current docs](https://aws-amplify.github.io/docs/ios/storage#restrict-access) and the goal is to bring them to parity (as long as it's correct).

One of the goals for Amplify iOS/Android GA is to review the documentation content, so we can address the missing content as part of that work.

Reply:
Opened an issue https://github.com/aws-amplify/docs/issues/1586 to not lose track.